study.md

```diff
@@ -151,5 +175,7 @@ Write some Ruby code that uses the Array `map` method and code blocks to
 duplicate this functionality.
 
 ```ruby
-# your answer here
+def numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
```

Comment:
You don't need a keyword to declare variables in Ruby.  The `def` keyword here is used to define a function and its presence in this particular line will cause an error.  Other than that, this is correct.